PubNubSwiftChatSDK.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,24 @@
+{
+  "originHash" : "2b7f8558564acf72b5e5be5d437803cf7d29d74b8d155e01d97b3918fb30f48e",
+  "pins" : [
+    {
+      "identity" : "kmp-chat",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/pubnub/kmp-chat",
+      "state" : {
+        "revision" : "ead1e5c07b5b14a1acb5c69315b33faa7d24b5c9",
+        "version" : "0.8.3-dev"
```

Comment:
I had to create draft/test release to upload the current KMP binary containing Message Draft to start wrapping it on my side. Therefore, I simulated a scenario in which our future release has `0.8.3-dev` tag for swift-chat-sdk. I created this tag temporarily on my branch, and later it will be replaced when we do "real" release. No additional work will be required, assuming that "real" release will also have `0.8.3-dev` tag. Then, SPM should be able to re-download again such dependency.

Perhaps we'll need to discuss the whole process and how to approach it for future releases. The ideal scenario for me would be releasing `kmp-chat` first (let's say, one iteration before). Then, I could work on existing stuff without doing manual work like those draft releases.

Reply:
Thank you for clarification.